app/frontend/src/features/profile/ProfilePage.tsx

```diff
@@ -34,6 +41,13 @@ export default function ProfilePage() {
         >
           Edit my place
         </ListItem>
+        <ListItem
```

Comment:
Not sure where the best place to put this, so this makes the most sense to me for now. I suspect we'd want to move this to the profile page eventually under some options button maybe?